tests/test_updater_consistent_snapshot.py

```diff
@@ -48,10 +48,7 @@ def _init_repo(
 
         # Init trusted root with the latest consistent_snapshot
         with open(os.path.join(self.metadata_dir, "root.json"), "bw") as f:
-            root = sim.download_bytes(
-                "https://example.com/metadata/2.root.json", 100000
-            )
-            f.write(root)
+            f.write(self.sim.signed_roots[1])
```

Comment:
As the above comment says latest,  the ``signed_roots`` is ordered, maybe would be more expressive use ``-1``?

```suggestion
            f.write(sim.signed_roots[-1])
```

Reply:
that makes sense... I seem to have added a bug as well with `self.`